### PR TITLE
new publish model

### PR DESCRIPTION
**Pull request by LukaszWiktor**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -341,6 +341,7 @@ try {
 
 Changelog
```
Comment (polok) [inline README.md]: This is a general comment. I think that we should create 'CHANGELOG.md' file under each our SDK repository and drop this section here.

**Comment by polok** [comment]
👍

File: README.md (hunk 1 above)
Comment (LukaszWiktor) [inline README.md]: Good point. Thanks @polok!

Comment (LukaszWiktor) [inline README.md]: I moved the changelog to CHANGELOG.md and left last 3 entries in README.md.

**Pull request merged**